src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+&-stars {
+  display: flex;
+
+  &-star {
+    width: 16px;
+    height: 16px;
+    flex-shrink: 0;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+    &:nth-child(-n + 4) {
```

Comment:
Here should still all the same amount of available modifiers like in previous task

Reply:
how

Reply:
> Here should still all the same amount of available modifiers like in previous task

and  why it wasn't asked 5 previous times